build_deploy.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+make build-deploy
```

Comment:
nit: should this be in our `hack` directory?

Reply:
Moved.